<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartMenu</class>
 <widget class="QWidget" name="CartMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CartMenu</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="orderButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>540</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">    QPushButton {
        background-color: #4BB34B;
        color: white;
           border-radius: 10px;               /* Радиус закругления углов */
    border: 2px solid #1E1F20;         /* Граница светло-серого цвета */
        padding: 10px 20px;
    }
    QPushButton:hover {
        background-color: #5fd35f;
    }
    QPushButton:pressed {
        background-color: #3fa53f;
    }</string>
     </property>
     <property name="text">
      <string>Оформить заказ</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>539</y>
       <width>571</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
    background-color: #313638;         /* Светлый серый фон */
    border-radius: 10px;               /* Радиус закругления углов */
    border: 2px solid #1E1F20;         /* Граница светло-серого цвета */
    padding: 10px;                     /* Внутренний отступ */
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="totalPriceLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #FFFAFB; /* Темно-серый текст */
}
</string>
      </property>
      <property name="text">
       <string>Общая стоимость:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #313638;        /* Синий фон */
    color: white;                     /* Белый текст */
        border-radius: 10px;               /* Радиус закругления углов */
    border: 2px solid #1E1F20;         /* Граница светло-серого цвета */
    padding: 5px 10px;                /* Отступы */
}
QPushButton:hover {
    background-color: #2980b9;        /* Более темный синий при наведении */
}
</string>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>781</width>
       <height>461</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>779</width>
        <height>459</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>-390</x>
         <y>-140</y>
         <width>549</width>
         <height>439</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>751</width>
         <height>441</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Корзина товаров</string>
     </property>
    </widget>
    <widget class="QPushButton" name="myOrderButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">    QPushButton {
        background-color: #313638;
        color: white;
        border: 1px solid #2e2e2e;
        border-radius: 8px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        border-color: #4e4e4e;
    }
    QPushButton:pressed {
        background-color: #1e1e1e;
}</string>
     </property>
     <property name="text">
      <string>Мои заказы</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
